README.md

```diff
@@ -147,4 +147,43 @@ See the [Ansible Community Guide](https://docs.ansible.com/ansible/latest/commun
 
 GNU General Public License v3.0 or later.
 
-See [LICENCE](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
+See [LICENSE](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
+
+<!--start collection content-->
+## Cliconf plugins
+Name | Description
+--- | ---
+[vyos.vyos.vyos](https://github.com/ansible-collections/vyos.vyos/blob/master/docs/vyos.vyos.vyos.rst)|Use vyos cliconf to run command on VyOS platform
+## Terminal plugins
```

Comment:
As discussed in yesterday's meeting, this needs to be removed because the terminal plugin doesn't have a doc string.